src/Analysis/GWASV2/Shared/SearchBar.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../../GWASUIApp/GWASUIApp.css';
+import { SearchOutlined } from '@ant-design/icons';
+
+const SearchBar = ({ searchTerm, handleSearch, fields = 'variable name' }) => (
```

Comment:
should it be `field` instead of `fields`?

Reply:
The name has been updated to `field` now